docs/latest/user/http-request-headers.md

```diff
@@ -221,49 +221,7 @@ $ curl -vvv --header "Host: headers.example" "http://${GATEWAY_HOST}:8080/get" -
 
 ## Combining Filters
 
-Headers can be added/set/removed in separate filters on the same HTTPRoute and they will all perform as expected
```

Comment:
```
for: "STDIN": error when patching "STDIN": admission webhook "validate.gateway.networking.k8s.io" denied the request: spec.rules[0].filters: Invalid value: "RequestHeaderModifier": cannot be used multiple times in the same rule
```
In v0.6.0, it looks like we can not use HeaderModifier in the same rule multiple times, so I remove this guide in request header modifier too.